### ISSUE TITLE
Adding Hindi translation.

### ISSUE THREAD

**Issue by yashk2000** [mit-cml/appinventor-sources#1581]
There are translations for many languages. I would like to add Hindi translations.

**Comment by yashk2000** [comment]
@ewpatton should I proceed with adding Hindi translations?

**Comment by ewpatton** [comment]
@SusanRatiLane Please advise @yashk2000 what needs to be done to integrate Hindi translations.

**Comment by yashk2000** [comment]
Thanks for the prompt response @ewpatton. @SusanRatiLane could you please guide me on how to proceed with adding Hindi translations?

**Comment by yashk2000** [comment]
I am planning to `extended property` of Hindi in this file. `appinventor/appengine/src/com/google/appinventor/YaClient.gwt.xml`.
Then adding Hindi in `appinventor/appengine/src/com/google/appinventor/client/languages.json` and `appinventor/blocklyeditor/ploverConfig.js`. To make Hindi appear in the language switch, I will also make changes in `appinventor/blocklyeditor/src/language_switch.js`. Then I will create a new folder for Hindi translations in `appinventor/blocklyeditor/src/msg` and  `appinventor/appengine/src/com/google/appinventor/client`. 

Should I proceed in this way?

**Comment by ewpatton** [comment]
@yashk2000 You have the right idea. If you want to get started just editing the files, first build App Inventor following the instructions in the readme. This will produce a directory called extras under appengine/build, and this will contain an English properties file for OdeMessages to serve as a starting point for the OdeMessages file. There's already an English translation under blocklyeditor that you can use as the basis for translating the blocks editor.

**Comment by SusanRatiLane** [comment]
As an alternative, we have a translation system using Google Translator Toolkit. Provided a Google account address, I can add you to that. If you prefer to submit the translation directly to Github, that is also fine. The result is the same -- Google Translator Toolkit is simply an attempt to simplify things for translators.

**Comment by yashk2000** [comment]
@ewpatton I have built the project and found the files which have to be changed. Thanks a lot for the help.

@SusanRatiLane I would like to try using the Google Translator Toolkit and make the translations using that. My Google account address is `yashsja@gmail.com`

**Comment by yashk2000** [comment]
@ewpatton I am a bit busy right now. Can I work on these translations later?

**Comment by SusanRatiLane** [comment]
Sure. Work as it as you have effort to spare.

Could you confirm that you got the invitation to Google Translator Toolkit?

**Comment by yashk2000** [comment]
Yes. I got the invitation to the Google Translator Toolkit. I even started with the translations. The Google Translator Toolkit makes things way easier than they otherwise would be. 

Thanks for allowing me to work on it later. I will complete the translations as soon as possible when I have time.
